import io.netty.resolver.dns.DnsResolveContext.DnsResolveContextException;
public class FunctionUse21180 {
public void funcUse() {
DnsResolveContextException dnsresolvecontextexception = new DnsResolveContextException();
}
}